flambe/runner/run.py

```diff
@@ -38,6 +38,10 @@ def main(args: argparse.Namespace) -> None:
         print(cl.RA(ASCII_LOGO))
         print(cl.BL(f"VERSION: {flambe.__version__}\n"))
 
+    if args.debug and args.cluster is not None:
```

Comment:
Are we sure that we want to deactivate all of the `debug` functionalities in remote execution?

Reply:
It's not clear to me how `debug` can work in a remote execution. Until we figure that out, I suggest `debug` just works locally for any runnable.

Reply:
sounds good